pom.xml

```diff
@@ -8,7 +8,7 @@
   <parent>
     <groupId>org.camunda</groupId>
     <artifactId>camunda-release-parent</artifactId>
-    <version>2.5</version>
+    <version>3.7</version>
```

Comment:
Where this version comes from? :thinking:

Reply:
it's the latest version of the `camunda-release-parent`. The commit message says:
`update camunda-release-parent@3.7 - required for deploying to both the Camunda Nexus and Maven Central`

The old version just doesn't support that.